src/reactjs/components/KVM/UI.tsx

```diff
@@ -63,7 +63,7 @@ export class RemoteDesktop extends React.Component<KVMProps, { kvmstate: number
 
   init (): void {
     const deviceUuid: string = this.props.deviceId != null ? this.props.deviceId : ''
-    const server: string = this.props.mpsServer != null ? this.props.mpsServer : ''
+    const server: string = this.props.mpsServer != null ? this.props.mpsServer.replace('http', 'ws') : ''
```

Comment:
Can you please make the same changes to src\reactjs\components\SerialOverLAN\Sol.tsx?